charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -71,174 +85,203 @@ data:
         Systemd_Filter  _SYSTEMD_UNIT=kubelet-monitor.service
 
   filter-kubernetes.conf: |-
+    [FILTER]
+        Name record_modifier
```

Comment:
Why similar filters are now added?

Reply:
These filters collect `systemd` logs.

Тhe idea is to separate for example: `docker-service` from `docker-monitor-service`. The old version of the filters did not work correctly because `docker-service` was catching logs from both services (`docker-service` and `docker-monitor-service`).